src/tlo/methods/alri.py

```diff
@@ -2650,6 +2654,16 @@ def _try_treatment(antibiotic_indicated: Tuple[str], oxygen_indicated: bool) ->
             oxygen_available = self._get_cons('Oxygen_Therapy')
             oxygen_provided = (oxygen_available and oxygen_indicated)
 
+            # todo: should equipment only be logged if consumables are available?
+            # If individual requires oxygen, log equipment
+            if oxygen_indicated:
+                self.EQUIPMENT.update({'Oxygen cylinder, with regulator', 'Nasal Prongs'})
+
+            # If individual requires intravenous antibiotics, log equipment
+            if antibiotic_indicated in ('1st_line_IV_antibiotics',
```

Comment:
```suggestion
            if antibiotic_provided in ('1st_line_IV_antibiotics',
```

Are you sure, this should be `antibiotic_indicated` and not `antibiotic_provided`?

Reply:
also updated this now